QuickDeployment/V2/QuickDeploymentDesktopReadme.md

```diff
@@ -90,6 +90,11 @@ If you would like to change the hostname of this VM, please do so **before** goi
 
 ### 3. Run the Set-QDEnvironment.ps1 Script
 
+>:memo: **Note**
```

Comment:
Blockquote `>` needs a space after it or it'll break the Markdown formatting.

Reply:
Done. This better be the last one :joy: